cms/envs/common.py

```diff
@@ -767,6 +767,11 @@
 EDLY_COOKIE_SECRET_KEY = 'EDLY-COOKIE-SECRET-KEY'
 EDLY_JWT_ALGORITHM = 'HS256'
 
+ALLOWED_SITE_CONFIGURATIONS_OVERRIDE = [
+    'BRANDING', 'COLORS', 'ALLOW_PUBLIC_ACCOUNT_CREATION', 'REGISTRATION_EXTRA_FIELDS',
+    'EMAILS_CONFIG', 'DJANGO_SETTINGS_OVERRIDE',
+]
+
```

Comment:
already being imported from lms common at the start.

Reply:
This is intentional. 
There can be some keys that are specific to cms and/or lms.  Due to this the dictionary is kept separate for both the services. 
With time ismaii zaida keys add hotii rahaingii that we want to enable to update from API.